lib/model/migrations/20200918-01-repair-publishedat-dates.js

```diff
@@ -0,0 +1,23 @@
+// Copyright 2020 ODK Central Developers
+// See the NOTICE file at the top-level directory of this distribution and at
+// https://github.com/opendatakit/central-backend/blob/master/NOTICE.
+// This file is part of ODK Central. It is subject to the license terms in
+// the LICENSE file found in the top-level directory of this distribution and at
+// https://www.apache.org/licenses/LICENSE-2.0. No part of ODK Central,
+// including this file, may be copied, modified, propagated, or distributed
+// except according to the terms contained in the LICENSE file.
+
+
+const up = (db) => db.raw(`
+update form_defs set "publishedAt"="loggedAt"
+  from (select (details->'newDefId')::int as "publishedId", "loggedAt"
```

Comment:
I don't understand the `int` cast here! That's jsonb and seems we just want it raw and used by the aliases?

Reply:
Maybe you meant to use `->>` which returns text? Then I guess that would cast both but that's ok because we don't care about `loggedAt`.

Reply:
huh? eventually we want to compare that value to `form_defs.id` which is an `int`. so we must cast it. but i can cast it below sure. figured making the data cleaner sooner would be better.

Reply:
oh i see, it fails in 9.6 this way.